chacractl/util.py

```diff
@@ -20,6 +21,7 @@ def load_config():
     config = get_config_path()
     if not config:
         return
+    sys.modules.pop('chacractl', None)
```

Comment:
since this is a test-only situation I would prefer this to be a fixture (should probably live in `tests/conftest.py`). That way, tests that require mangling of the configuration can do so by requesting it:

    import sys
    
    
    @pytest.fixture(scope='session')
    def clear_config(request):
        sys.modules.pop('chacractl', None)
        chacractl.util.load_config()